HW-10/homework10.py

```diff
@@ -0,0 +1,14 @@
+def MyDec(func):
+    from datetime import datetime
+
+    def wrap(*args, **kwargs):
+        time = datetime.now()
+        func(*args, **kwargs)
```

Comment:
assign the result that returns this func to some value and return it after the last print to be able to decorate functions that return some value

Reply:
The changes you made is not something I was asking for. I was asking to assign the result to some value:
`res = func(*args, **kwargs)`
And return it after last print:
```
res = func(*args, **kwargs)
print(f'Open function: {func} \n Time: {time}')
return res
```

But you instead added something that breaks your decorator, and now it doesn't work as it should